lib/utils/getServerlessConfigFile.js

```diff
@@ -46,28 +53,58 @@ const getServerlessConfigFilePath = serverless => {
   );
 };
 
-const handleJsConfigFile = jsConfigFile =>
+const resolveTsNode = serviceDir => {
+  const resolveModuleRealPath = (...args) =>
+    resolveModulePath(...args).then(({ realPath }) => realPath);
+
+  const createOnError = cb => error => {
```

Comment:
Let's rename it to `ifNotFoundContinueWith` (at least `createOnError` doesn't reflect what this util does)

Reply:
done